internal/controller/pkg/revision/establisher_test.go

```diff
@@ -402,7 +403,12 @@ func TestAPIEstablisherEstablish(t *testing.T) {
 			if diff := cmp.Diff(tc.want.err, err, test.EquateErrors()); diff != "" {
 				t.Errorf("\n%s\ne.Check(...): -want error, +got error:\n%s", tc.reason, diff)
 			}
-			if diff := cmp.Diff(tc.want.refs, refs, test.EquateErrors()); diff != "" {
+			trans := cmp.Transformer("Sort", func(refs []xpv1.TypedReference) []xpv1.TypedReference {
+				out := append([]xpv1.TypedReference(nil), refs...) // Copy input to avoid mutating it
+				sort.SliceStable(out, func(i, j int) bool { return out[i].Name < out[j].Name })
+				return out
+			})
```

Comment:
Could you use the built in SortSlices option instead?

https://pkg.go.dev/github.com/google/go-cmp/cmp/cmpopts#SortSlices

Reply:
nice! done!